bindings/ocaml/SharedDefs.ml

```diff
@@ -4,32 +4,39 @@ open Unsigned
 module type Buffer = sig
   type t
   type buf
-  val size_uint32 : t -> uint32
-  val ctypes_buf : t -> buf
-  val size : t -> int
-  val equal : t -> t -> bool
-  val disjoint : t -> t -> bool
-  val sub : t -> int -> int -> t
-  val z_compare : t -> Z.t -> int
+  val empty: bytes
+  val size_uint32 : bytes -> uint32
+  val ctypes_buf : bytes -> buf
+  val size : bytes -> int
+  val equal : bytes -> bytes -> bool
+  val make : int -> bytes
+  val disjoint : bytes -> bytes -> bool
+  val sub : bytes -> int -> int -> bytes
+  val z_compare : bytes -> Z.t -> int
 end
+(** Abstract representation of buffers *)
 
 module CBytes : Buffer with type t = Bytes.t and type buf = Bytes.t Ctypes.ocaml = struct
   type t = Bytes.t
   type buf = Bytes.t Ctypes.ocaml
+  let empty = Bytes.empty
   let size_uint32 b = Unsigned.UInt32.of_int (Bytes.length b)
   let ctypes_buf = Ctypes.ocaml_bytes_start
   let size = Bytes.length
   let equal = Bytes.equal
+  let make l = Bytes.make l '\x00'
   let disjoint b1 b2 = b1 != b2
   let sub = Bytes.sub
   let z_compare b z = Z.compare (Z.of_bits (Bytes.to_string b)) z
 end
+(** Representation of [Bytes.t] buffers *)
 
-(* VD: temporarely disable, eliminate dependency on bigstring *)
+(* VD: temporarily disable, eliminate dependency on bigstring *)
```

Comment:
Can we remove that commented code now ?

Reply:
I guess we could. I wanted to keep compatibility with Bigstring in case we at some point decide to use that instead of / in addition to Bytes and I kept this commented code around to give some context to the extra complexity that results from this in the code. Thoughts @msprotz ?

Reply:
No strong opinion. I thought we ruled out Bigstring a while back? I'm generally against keeping dead code in comments, since there's always version control. You could leave a comment that says "for a version of this built on `BigString`, see 4c1ee47"